packages/mask/src/plugins/Trader/SNSAdaptor/trader/Trader.tsx

```diff
@@ -169,6 +176,18 @@ export function Trader(props: TraderProps) {
 
     // Query the balance of native tokens on target chain
     useAsync(async () => {
+        if (!wallet) {
+            dispatchTradeStore({
+                type: AllProviderTradeActionType.UPDATE_INPUT_TOKEN_BALANCE,
+                balance: '0',
+            })
+
+            dispatchTradeStore({
+                type: AllProviderTradeActionType.UPDATE_OUTPUT_TOKEN_BALANCE,
+                balance: '0',
+            })
+        }
```

Comment:
Does it need to early return?
```suggestion
        if (!wallet) {
            dispatchTradeStore({
                type: AllProviderTradeActionType.UPDATE_INPUT_TOKEN_BALANCE,
                balance: '0',
            })

            dispatchTradeStore({
                type: AllProviderTradeActionType.UPDATE_OUTPUT_TOKEN_BALANCE,
                balance: '0',
            })
            return
        }
```